types/aws-param-store/aws-param-store-tests.ts

```diff
@@ -12,6 +12,7 @@ import {
 
 declare let bool: boolean;
 declare let query: ParameterQuery;
+declare let path: string;
```

Comment:
I don't think path is needed as `SSM.Types.PSParameterName` is string in the end.